mysql/mysqld.cnf

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2014, 2016, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I don't think the copyright notice is necessary

Reply:
Ok I will remove them.